website/docs/r/elasticache_serverless.html.markdown

```diff
@@ -0,0 +1,139 @@
+---
+subcategory: "ElastiCache"
+layout: "aws"
+page_title: "AWS: aws_elasticache_serverless"
+description: |-
+  Provides an ElastiCache Serverless Cluster resource.
+---
+
+# Resource: aws_elasticache_serverless
+
+Provides an ElastiCache Serverlesss Cluster resource which manages memcache or redis.
+
+## Example Usage
+
+### Memcached Serverless
+
+```terraform
+resource "aws_elasticache_serverless" "example" {
+  engine                = "memcached"
+  serverless_cache_name = "example"
+  cache_usage_limits {
+    data_storage {
+      maximum = 10
+      unit    = "GB"
+    }
+    ecpu_per_second {
+      maximum = 5
+    }
+  }
+  description          = "Test Server"
+  kms_key_id           = aws_kms_key.test.arn
+  major_engine_version = "1.6"
+  security_group_ids   = [aws_security_group.test.id]
+  subnet_ids           = aws_subnet.test[*].id
+}
+```
+
+### Redis Serverless
+
+```terraform
+resource "aws_elasticache_serverless" "example" {
+  engine                = "redis"
+  serverless_cache_name = "example"
+  cache_usage_limits {
+    data_storage {
+      maximum = 10
+      unit    = "GB"
+    }
+    ecpu_per_second {
+      maximum = 5
+    }
+  }
+  daily_snapshot_time      = "09:00"
+  description              = "Test Server"
+  kms_key_id               = aws_kms_key.test.arn
+  major_engine_version     = "1.6"
```

Comment:
should be something like 7.1 imo

Reply:
Yeah -- you are right, its a copy pasta error butt fixed it now. I ran the testts with verssion 7 and amended to the same. For some reason when I ran the tests it said engine version 7.1 is not supported.

Reply:
@meetreks i believe it should be engine version 7. Was trying out awscc provider elasticache serverless resource earlier and it only accepted 7 not 7.1 for major engine version.